app/assets/javascripts/components/annotations/annotation_marker.ts

```diff
@@ -34,6 +34,10 @@ export class AnnotationMarker extends LitElement {
         if (["error", "warning", "info"].includes(annotation.type)) {
             return `
                 text-decoration: wavy underline ${AnnotationMarker.colors[annotation.type]} 1px;
+                -webkit-text-decoration: underline;
+                -webkit-text-decoration-color: ${AnnotationMarker.colors[annotation.type]};
+                -webkit-text-decoration-thickness: 1px;
+                -webkit-text-decoration-style: wavy;
```

Comment:
I don't have a safari browser directly available, but are the prefixes necessary here?

According to what I could find, the shorthand `text-decoration` property needs a prefix in Safari, but the individual ones don't. So, in theory, we should be able the following and cover all browsers (and also removing the line above the changes):

```suggestion
                text-decoration-line: underline;
                text-decoration-color: ${AnnotationMarker.colors[annotation.type]};
                text-decoration-thickness: 1px;
                text-decoration-style: wavy;
```